mmcv/version.py

```diff
@@ -32,6 +30,6 @@ def parse_version_info(version_str: str, length: int = 4) -> tuple:
     return tuple(release)
 
 
-version_info = parse_version_info(__version__)
+version_info = tuple([int(x) for x in __version__.split('.')])
```

Comment:
```suggestion
version_info = tuple(int(x) for x in __version__.split('.'))
```
In addition, if `__version__` contains non-numeric string like `1.3.10.dev`, `int(x)` will throw an error